### PR TITLE
new file: mv.go

### PR DESCRIPTION
**Pull request by rhiguita**

### CODE REVIEW

File: src/cmds/mv/mv.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	"os"
+)
+
+func main() {
+	flag.Parse()
+	a := flag.Args()
+	if len(a) < 2 {
+		fmt.Printf("mv - missing file operand\n")
+	} else {
+		error := os.Rename(a[0], a[1])
```
Comment (mirtchovski) [inline src/cmds/mv/mv.go]: please don't shadow "error" here. the idiomatic way to do it is to say:

err := ...
if err != nil {
    // handle error
}

**Comment by mirtchovski** [comment]
in general this needs a little bit more work: the original knew to check if the last argument was a directory, then it would "move" all the preceding arguments to that directory, taking more than one filename for the purpose if needed.

in general, the Plan 9 mv.c source file is a good example of the expected behaviour:

http://plan9.bell-labs.com/sources/plan9/sys/src/cmd/mv.c

**Comment by mirtchovski** [comment]
by the way, do you have a machine with an incorrect date set on it? it says that you're in the same timezone as rminnich, but your patches come from one day in the future. please adjust:

```
commit 19e15b1b6d8f27a59ece34f75ddb680cd14e4452
Merge: d0eb523 328c337
Author: ron minnich <rminnich@gmail.com>
Date:   Tue Aug 25 18:31:33 2015 -0700

    Merge pull request #14 from rhiguita/master
    
    new file: mv.go

commit 328c3370aeca9e86cb8f8db1225f3eef658ca132
Author: Beletti <rhiguita@gmail.com>
Date:   Wed Aug 26 00:06:47 2015 -0700

    fix mv.go

commit 08da13a83dc824193c158ba289e51a8f92ddda6a
Author: Beletti <rhiguita@gmail.com>
Date:   Tue Aug 25 22:37:01 2015 -0700

    new file: mv.go

```

**Comment by rhiguita** [comment]
Yes, my date was wrong. Fixed! Sorry!

2015-08-25 19:07 GMT-07:00 mirtchovski <notifications@github.com>:

File: src/cmds/mv/mv.go (hunk 1 above)
Comment (rhiguita) [inline src/cmds/mv/mv.go]: Ok, I will do a check to each error.

2015-08-25 18:59 GMT-07:00 mirtchovski <notifications@github.com>:

Comment (mirtchovski) [inline src/cmds/mv/mv.go]: just don't name it `error`, that's the name of the error package. name it `err`

**Comment by rhiguita** [comment]
Ok, I will take a look in Plan 9 mv.c source file.

2015-08-25 19:01 GMT-07:00 mirtchovski <notifications@github.com>:

File: src/cmds/mv/mv.go (hunk 1 above)
Comment (rhiguita) [inline src/cmds/mv/mv.go]: Ok, thank you again!

2015-08-25 19:18 GMT-07:00 mirtchovski <notifications@github.com>:

**Pull request merged**